Form-Controls/index.html

```diff
@@ -17,11 +17,46 @@ <h1>Product Pick</h1>
         <!-- 
           try writing out the requirements first as comments
         this will also help you fill in your PR message later-->
+
+        <!--What is the customer's name?-->
+        <label for="fname"> First Name: </label>
+        <input type="text" id="fname" name="fname" minlength="2" /><br><br>
+        <label for="lname"> Last Name: </label>
+        <input type="text" id="lname" name="lname" minlength="2"/><br><br>
+        <!--What is the customer's email?-->
+        <label for="email">Enter your email: </label>
+        <input type="email" id="email" name="email" required/><br><br>
+
+        <!--What colour should this t-shirt be? I must give 3 options. 
+        How will I make sure they don't pick other colours? -->
+        
+        <label for="colours" >T-shirt colours: </label>
+        <select id="colours" name="colours">
+          <option disabled selected value> -- select colour -- </option>
+          <option value="blue">Blue</option>
+          <option value="green">Green</option>
+          <option value="red">Red</option>
+        </select><br><br>
+        
+        <!--What size does the customer want? I must give the 
+        following 6 options: XS, S, M, L, XL, XXL-->
+        <label for="size"> T-shirt size: </label>
+        <select id="size" name="size">
+          <option disabled selected value> -- select size --</option>
+          <option> XS </option>
+          <option> S </option>
+          <option> M </option>
+          <option> L </option>
+          <option> XL </option>
+          <option> XXL </option>
+      </select><br>
+        <br><input type="reset" value="Reset"/>
+        <input type="submit" value="Submit"/>
```

Comment:
Nice to see you added reset button, you can also check other types for input tag from here https://www.w3schools.com/html/html_form_input_types.asp